components/amorphic/HISTORY.md

```diff
@@ -1,5 +1,5 @@
-## 8.0.0
-* Updating for injectable config, and config is now in supertype
+## 7.1.3
```

Comment:
Did you mean to delete the 8.0.0 history, or should we have both?

Reply:
Yeah I would like to delete